app/src/main/java/com/usepace/android/messagingcenter/clients/connection_client/MessageCenter.java

```diff
@@ -44,6 +44,43 @@ public static void connect(Context context, ConnectionRequest connectionRequest,
         }
     }
 
+
+    /**
+     *
+     * @param context
+     *
+     *
+     */
+    public static void reInitClient(Context context) {
+        try {
+            client().getClient(LAST_CLIENT).reInit(context);
+        }
+        catch (MessageCenterException e) {
+            //connectionInterface.onMessageCenterConnectionError(e.getCode(), e);
+        }
+    }
+
+
+
+
+    /**
+     *
+     *
+     *
+     *
+     */
+    public static void reConnect() {
```

Comment:
should not be accessible by app

Reply:
resolved by adding a extension class, as discussed